.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+
+name: 'deploy'
+
+# yamllint disable-line rule:truthy
+on:
+  tags:
+    - release
```

Comment:
Will double-check on the spec. Pretty sure I saw it had support for tags, but its not something I have much experience with.

Regarding whether it should be `master`, the though process was that creating a tag is fairly simple to do via the UI and would allow a bit more control on when a release to prod occurs. If we're going to strongly require folks to create PRs that get reviewed and have to be approved and therefore `master` is always production ready, I'm fine with that. I just wanted to get something that was easier to control in first so that we can test the flow with a staging version on our production server.

Reply:
Currently every PR requires 1 approval to be merged so hopefully they are production ready. Regardless if we want to be safe, I would set it up so that only on push to `master`, the dokku deploys the updates to the staging website. Then have a separate job that on a specific tag, will deploy that commit to the production website. So you can see what's on master in the staging website and then when satisfied can manually triger a  deploy to prod.

Reply:
I can also implement this if you are busy

Reply:
It sounds like we're currently on the same page. This particular workflow is intended to be an experiment to ensure the configurations are correct for the staging environment. It should then be trivial to make an updated version for production.

Reply:
Ok I see, this isn't the final version.